test/common/router/scoped_rds_test.cc

```diff
@@ -930,7 +971,516 @@ route_configuration_name: foo_routes
   EXPECT_EQ(2UL,
             server_factory_context_.scope_.counter("foo.scoped_rds.foo_scoped_routes.config_reload")
                 .value());
-  EXPECT_EQ(getScopedRouteMap().size(), 2);
+  EXPECT_EQ(2UL, server_factory_context_.scope_
+                     .gauge("foo.scoped_rds.foo_scoped_routes.all_scopes",
+                            Stats::Gauge::ImportMode::Accumulate)
+                     .value());
+}
+
+// Compare behavior of a lazy scope and an eager scope scopes that share that same route
+// configuration. roue config of on demand scope shouldn't be loaded.
```

Comment:
`. Route config of the on-demand scope shouldn't be loaded.`

Reply:
Sorry, typo.